examples/secured-client-with-oauth2/secured_client_with_oauth2.description

```diff
@@ -0,0 +1,7 @@
+// A secured client with oauth2 authentication is used to connect to a
```

Comment:
Change 'oauth2' to 'OAuth2'. Let's fix everywhere for consistency.

Reply:
Fixed.